### ISSUE TITLE
GePhi freezes after successfully importing dynamic gexf file

### ISSUE THREAD

**Issue by Haonan-Zhou** [gephi/gephi#1566]
I'm using GePhi 0.9.1 on OS X to import gexf file into the workspace. I test three gexf files containing dynamic graphs. 

File 1 is the gexf file saved from the built-in dynamic graph example. Load into GePhi, everything is fine.

File 2 comes from Kikohs's answer: [http://stackoverflow.com/questions/34646956/making-dynamic-graphs-in-pythons-networkx](url). Load into GePhi, the usual import screen appears, which seems to suggest the gexf file is fine. 
<img width="635" alt="screen shot 2016-09-03 at 11 44 30 pm" src="https://cloud.githubusercontent.com/assets/16145368/18228933/619feca0-7230-11e6-89c9-a6bf647f5841.png">

However, GePhi freezes after importing the graph, although the bottom-left corner suggests "1 sources successfully imported". There's the error icon on the bottom-right corner.

File 3 is my own dynamic gexf file. Significantly larger than File 2. Encounter the same problem. 

I also tested File 3 on a Windows 7 machine. While the file can be loaded and GePhi did not freeze. It cannot calculate dynamic graph statistics, nor can it be saved as a .gephi project file (can be saved, actually, but the size is 0KB). In addition, the bottom right corner suggests "null pointer exception". 

For reference: the key structural difference between File 1 and File 2,3 is that File 1 does not have dynamic edge attributes, while both File 2 and 3 have dynamic edge attributes, in particular the weight attributes is overridden using 

`<attributes class="edge" mode="dynamic">
   <attribute id="weight" title="Weight" type="float"/>
</attributes>`

**Comment by duncdrum** [comment]
@Haonan-Zhou the error message on the import screen points to the use of gexf1.2 with ```@endopen``` attributes on the edges. Like so:
```xml
<edge source="n0" target="n1">
        <attvalues>
          <attvalue for="weight" value="3.0" start="2010.0" endopen="2012.0"></attvalue>
...
```
Gephi 0.9.1 uses gexf 1.3 which no longer supports ```@endopen``` so you need to either use these graphs from inside gephi 0.8 using gexf 1.2, or you need to adjust the schema to point to gexf1.3 and change the name of the endopen to the appropriate format.

**Comment by Haonan-Zhou** [comment]
Thanks for the reply. Although File 2 does use GEXF 1.2, File 3 (the larger one) uses GEXF 1.3 with time representation specified as timestamp.

**Comment by duncdrum** [comment]
@Haonan-Zhou could you maybe share a sample from file 3, so we can try to reproduce the error. I m loosing track. What does the error message say? Where and how is timestamp specified in the file?

**Comment by Haonan-Zhou** [comment]
I attach a sample gexf from file 3 here, which only contain two nodes and one edge. The original graph contains 200+ nodes and 3000+ edges. Opening this sample encounters the same problem. 

The code is quite long... sorry.

On Mac, I can't recover the error message since it immediately freezes after I click "OK" in the import window.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<gexf xmlns="http://www.gexf.net/1.3" version="1.3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.gexf.net/1.3 http://www.gexf.net/1.3/gexf.xsd">
  <meta lastmodifieddate="2016-09-03">
    <creator>Haonan Zhou</creator>
    <description></description>
  </meta>
  <graph mode="dynamic" defaultedgetype="directed" timeformat="integer" timerepresentation="timestamp">
    <attributes class="node" mode="static">
      <attribute id="IFS" title="IFS" type="integer"/>
      <attribute id="region" title="region" type="integer"/>
      <attribute id="subregion" title="subregion" type="integer"/>
      <attribute id="lat" title="lat" type="double"/>
      <attribute id="lng" title="lng" type="double"/>
    </attributes>
    <attributes class="node" mode="dynamic">
      <attribute id="U_i" title="U_i" type="double"/>
      <attribute id="pcrisis_i" title="pcrisis_i" type="double"/>
      <attribute id="U_j" title="U_j" type="double"/>
      <attribute id="gdp_j" title="gdp_j" type="double"/>
    </attributes>
    <attributes class="edge" mode="static">
      <attribute id="SourceIFS" title="SourceIFS" type="integer"/>
      <attribute id="TargetIFS" title="TargetIFS" type="integer"/>
      <attribute id="homelender" title="homelender" type="integer"/>
      <attribute id="hostlender" title="hostlender" type="integer"/>
    </attributes>
    <attributes class="edge" mode="dynamic">
      <attribute id="weight" title="Weight" type="float"/>
      <attribute id="P_i" title="P_i" type="double"/>
      <attribute id="P_j" title="P_j" type="double"/>
    </attributes>
    <nodes>
      <node id="1" label="A">
        <spells>
          <spell timestamp="1"/>
          <spell timestamp="2"/>
          <spell timestamp="3"/>
          <spell timestamp="4"/>
          <spell timestamp="5"/>
          <spell timestamp="6"/>
          <spell timestamp="7"/>
          <spell timestamp="8"/>
          <spell timestamp="9"/>
          <spell timestamp="10"/>
          <spell timestamp="11"/>
          <spell timestamp="12"/>
          <spell timestamp="13"/>
          <spell timestamp="14"/>
          <spell timestamp="15"/>
          <spell timestamp="16"/>
          <spell timestamp="17"/>
          <spell timestamp="18"/>
          <spell timestamp="19"/>
          <spell timestamp="20"/>
          <spell timestamp="21"/>
          <spell timestamp="22"/>
          <spell timestamp="23"/>
          <spell timestamp="24"/>
          <spell timestamp="25"/>
          <spell timestamp="26"/>
          <spell timestamp="27"/>
          <spell timestamp="28"/>
          <spell timestamp="29"/>
          <spell timestamp="30"/>
          <spell timestamp="31"/>
          <spell timestamp="32"/>
        </spells>
        <attvalues>
          <attvalue for="IFS" value="1"/>
          <attvalue for="region" value="19"/>
          <attvalue for="subregion" value="21"/>
          <attvalue for="lat" value="39.622002"/>
          <attvalue for="lng" value="-98.606003"/>
          <attvalue for="U_i" value="786.77295" timestamp="1"/>
          <attvalue for="U_i" value="845.96387" timestamp="2"/>
          <attvalue for="U_i" value="878.0719" timestamp="3"/>
          <attvalue for="U_i" value="903.19287" timestamp="4"/>
          <attvalue for="U_i" value="1026.1819" timestamp="5"/>
          <attvalue for="U_i" value="1154.7888" timestamp="6"/>
          <attvalue for="U_i" value="1118.6077" timestamp="7"/>
          <attvalue for="U_i" value="1129.0095" timestamp="8"/>
          <attvalue for="U_i" value="1155.3529" timestamp="9"/>
          <attvalue for="U_i" value="1113.2354" timestamp="10"/>
          <attvalue for="U_i" value="992.99377" timestamp="11"/>
          <attvalue for="U_i" value="876.81451" timestamp="12"/>
          <attvalue for="U_i" value="1827.5206" timestamp="13"/>
          <attvalue for="U_i" value="1770.4691" timestamp="14"/>
          <attvalue for="U_i" value="2171.1702" timestamp="15"/>
          <attvalue for="U_i" value="2039.889" timestamp="16"/>
          <attvalue for="U_i" value="2136.0571" timestamp="17"/>
          <attvalue for="U_i" value="2078.6763" timestamp="18"/>
          <attvalue for="U_i" value="1757.3998" timestamp="19"/>
          <attvalue for="U_i" value="1774.8673" timestamp="20"/>
          <attvalue for="U_i" value="1905.3054" timestamp="21"/>
          <attvalue for="U_i" value="1961.1185" timestamp="22"/>
          <attvalue for="U_i" value="1790.9567" timestamp="23"/>
          <attvalue for="U_i" value="1735.375" timestamp="24"/>
          <attvalue for="U_i" value="1802.5848" timestamp="25"/>
          <attvalue for="U_i" value="1780.9672" timestamp="26"/>
          <attvalue for="U_i" value="1917.5984" timestamp="27"/>
          <attvalue for="U_i" value="2014.1315" timestamp="28"/>
          <attvalue for="U_i" value="2061.5994" timestamp="29"/>
          <attvalue for="U_i" value="1982.3928" timestamp="30"/>
          <attvalue for="U_i" value="2103.9563" timestamp="31"/>
          <attvalue for="U_i" value="1952.5347" timestamp="32"/>
          <attvalue for="pcrisis_i" value="0.094339602" timestamp="1"/>
          <attvalue for="pcrisis_i" value="0.094339602" timestamp="2"/>
          <attvalue for="pcrisis_i" value="0.094339602" timestamp="3"/>
          <attvalue for="pcrisis_i" value="0.094339602" timestamp="4"/>
          <attvalue for="pcrisis_i" value="0.1636364" timestamp="5"/>
          <attvalue for="pcrisis_i" value="0.1636364" timestamp="6"/>
          <attvalue for="pcrisis_i" value="0.1636364" timestamp="7"/>
          <attvalue for="pcrisis_i" value="0.1636364" timestamp="8"/>
          <attvalue for="pcrisis_i" value="0.094339602" timestamp="9"/>
          <attvalue for="pcrisis_i" value="0.094339602" timestamp="10"/>
          <attvalue for="pcrisis_i" value="0.094339602" timestamp="11"/>
          <attvalue for="pcrisis_i" value="0.094339602" timestamp="12"/>
          <attvalue for="pcrisis_i" value="0.078125" timestamp="13"/>
          <attvalue for="pcrisis_i" value="0.078125" timestamp="14"/>
          <attvalue for="pcrisis_i" value="0.078125" timestamp="15"/>
          <attvalue for="pcrisis_i" value="0.078125" timestamp="16"/>
          <attvalue for="pcrisis_i" value="0.042307701" timestamp="17"/>
          <attvalue for="pcrisis_i" value="0.042307701" timestamp="18"/>
          <attvalue for="pcrisis_i" value="0.042307701" timestamp="19"/>
          <attvalue for="pcrisis_i" value="0.042307701" timestamp="20"/>
          <attvalue for="pcrisis_i" value="0.050228301" timestamp="21"/>
          <attvalue for="pcrisis_i" value="0.050228301" timestamp="22"/>
          <attvalue for="pcrisis_i" value="0.050228301" timestamp="23"/>
          <attvalue for="pcrisis_i" value="0.050228301" timestamp="24"/>
          <attvalue for="pcrisis_i" value="0.066137597" timestamp="25"/>
          <attvalue for="pcrisis_i" value="0.066137597" timestamp="26"/>
          <attvalue for="pcrisis_i" value="0.066137597" timestamp="27"/>
          <attvalue for="pcrisis_i" value="0.066137597" timestamp="28"/>
          <attvalue for="pcrisis_i" value="0.053061198" timestamp="29"/>
          <attvalue for="pcrisis_i" value="0.053061198" timestamp="30"/>
          <attvalue for="pcrisis_i" value="0.053061198" timestamp="31"/>
          <attvalue for="pcrisis_i" value="0.053061198" timestamp="32"/>
          <attvalue for="U_j" value="2569.6248" timestamp="1"/>
          <attvalue for="U_j" value="2691.3628" timestamp="2"/>
          <attvalue for="U_j" value="2836.8645" timestamp="3"/>
          <attvalue for="U_j" value="3128.5698" timestamp="4"/>
          <attvalue for="U_j" value="3186.1814" timestamp="5"/>
          <attvalue for="U_j" value="3303.4639" timestamp="6"/>
          <attvalue for="U_j" value="3529.4282" timestamp="7"/>
          <attvalue for="U_j" value="3444.1462" timestamp="8"/>
          <attvalue for="U_j" value="3619.7063" timestamp="9"/>
          <attvalue for="U_j" value="3436.2295" timestamp="10"/>
          <attvalue for="U_j" value="3229.2634" timestamp="11"/>
          <attvalue for="U_j" value="2898.3125" timestamp="12"/>
          <attvalue for="U_j" value="2739.9829" timestamp="13"/>
          <attvalue for="U_j" value="2763.7344" timestamp="14"/>
          <attvalue for="U_j" value="2520.5859" timestamp="15"/>
          <attvalue for="U_j" value="2369.2017" timestamp="16"/>
          <attvalue for="U_j" value="2457.8242" timestamp="17"/>
          <attvalue for="U_j" value="2451.6357" timestamp="18"/>
          <attvalue for="U_j" value="2607.8955" timestamp="19"/>
          <attvalue for="U_j" value="2519.4822" timestamp="20"/>
          <attvalue for="U_j" value="2566.8467" timestamp="21"/>
          <attvalue for="U_j" value="2562.4519" timestamp="22"/>
          <attvalue for="U_j" value="2458.9771" timestamp="23"/>
          <attvalue for="U_j" value="2348.5559" timestamp="24"/>
          <attvalue for="U_j" value="2414.354" timestamp="25"/>
          <attvalue for="U_j" value="2337.842" timestamp="26"/>
          <attvalue for="U_j" value="2378.6689" timestamp="27"/>
          <attvalue for="U_j" value="2332.5542" timestamp="28"/>
          <attvalue for="U_j" value="2313.2764" timestamp="29"/>
          <attvalue for="U_j" value="2129.7036" timestamp="30"/>
          <attvalue for="U_j" value="2174.9702" timestamp="31"/>
          <attvalue for="U_j" value="2416.1311" timestamp="32"/>
          <attvalue for="gdp_j" value="13857.9" timestamp="1"/>
          <attvalue for="gdp_j" value="13857.9" timestamp="2"/>
          <attvalue for="gdp_j" value="13857.9" timestamp="3"/>
          <attvalue for="gdp_j" value="13857.9" timestamp="4"/>
          <attvalue for="gdp_j" value="14480.35" timestamp="5"/>
          <attvalue for="gdp_j" value="14480.35" timestamp="6"/>
          <attvalue for="gdp_j" value="14480.35" timestamp="7"/>
          <attvalue for="gdp_j" value="14480.35" timestamp="8"/>
          <attvalue for="gdp_j" value="14720.25" timestamp="9"/>
          <attvalue for="gdp_j" value="14720.25" timestamp="10"/>
          <attvalue for="gdp_j" value="14720.25" timestamp="11"/>
          <attvalue for="gdp_j" value="14720.25" timestamp="12"/>
          <attvalue for="gdp_j" value="14417.95" timestamp="13"/>
          <attvalue for="gdp_j" value="14417.95" timestamp="14"/>
          <attvalue for="gdp_j" value="14417.95" timestamp="15"/>
          <attvalue for="gdp_j" value="14417.95" timestamp="16"/>
          <attvalue for="gdp_j" value="14958.3" timestamp="17"/>
          <attvalue for="gdp_j" value="14958.3" timestamp="18"/>
          <attvalue for="gdp_j" value="14958.3" timestamp="19"/>
          <attvalue for="gdp_j" value="14958.3" timestamp="20"/>
          <attvalue for="gdp_j" value="15533.825" timestamp="21"/>
          <attvalue for="gdp_j" value="15533.825" timestamp="22"/>
          <attvalue for="gdp_j" value="15533.825" timestamp="23"/>
          <attvalue for="gdp_j" value="15533.825" timestamp="24"/>
          <attvalue for="gdp_j" value="16244.575" timestamp="25"/>
          <attvalue for="gdp_j" value="16244.575" timestamp="26"/>
          <attvalue for="gdp_j" value="16244.575" timestamp="27"/>
          <attvalue for="gdp_j" value="16244.575" timestamp="28"/>
          <attvalue for="gdp_j" value="16799.7" timestamp="29"/>
          <attvalue for="gdp_j" value="16799.7" timestamp="30"/>
          <attvalue for="gdp_j" value="16799.7" timestamp="31"/>
          <attvalue for="gdp_j" value="16799.7" timestamp="32"/>
        </attvalues>
      </node>
      <node id="68" label="B">
        <spells>
          <spell timestamp="1"/>
          <spell timestamp="2"/>
          <spell timestamp="3"/>
          <spell timestamp="4"/>
          <spell timestamp="5"/>
          <spell timestamp="6"/>
          <spell timestamp="7"/>
          <spell timestamp="8"/>
          <spell timestamp="9"/>
          <spell timestamp="10"/>
          <spell timestamp="11"/>
          <spell timestamp="12"/>
          <spell timestamp="13"/>
          <spell timestamp="14"/>
          <spell timestamp="15"/>
          <spell timestamp="16"/>
          <spell timestamp="17"/>
          <spell timestamp="18"/>
          <spell timestamp="19"/>
          <spell timestamp="20"/>
          <spell timestamp="21"/>
          <spell timestamp="22"/>
          <spell timestamp="23"/>
          <spell timestamp="24"/>
          <spell timestamp="25"/>
          <spell timestamp="26"/>
          <spell timestamp="27"/>
          <spell timestamp="28"/>
          <spell timestamp="29"/>
          <spell timestamp="30"/>
          <spell timestamp="31"/>
          <spell timestamp="32"/>
        </spells>
        <attvalues>
          <attvalue for="IFS" value="2"/>
          <attvalue for="region" value="19"/>
          <attvalue for="subregion" value="29"/>
          <attvalue for="lat" value="19.313999"/>
          <attvalue for="lng" value="-81.197998"/>
          <attvalue for="U_j" value="368.55499" timestamp="1"/>
          <attvalue for="U_j" value="403.72302" timestamp="2"/>
          <attvalue for="U_j" value="423.45801" timestamp="3"/>
          <attvalue for="U_j" value="463.871" timestamp="4"/>
          <attvalue for="U_j" value="495.29001" timestamp="5"/>
          <attvalue for="U_j" value="539.69501" timestamp="6"/>
          <attvalue for="U_j" value="595.867" timestamp="7"/>
          <attvalue for="U_j" value="621.328" timestamp="8"/>
          <attvalue for="U_j" value="609.97797" timestamp="9"/>
          <attvalue for="U_j" value="615.32404" timestamp="10"/>
          <attvalue for="U_j" value="624.64099" timestamp="11"/>
          <attvalue for="U_j" value="514.60901" timestamp="12"/>
          <attvalue for="U_j" value="614.62402" timestamp="13"/>
          <attvalue for="U_j" value="584.73303" timestamp="14"/>
          <attvalue for="U_j" value="582.54901" timestamp="15"/>
          <attvalue for="U_j" value="550.67999" timestamp="16"/>
          <attvalue for="U_j" value="580.16602" timestamp="17"/>
          <attvalue for="U_j" value="587.13696" timestamp="18"/>
          <attvalue for="U_j" value="589.67297" timestamp="19"/>
          <attvalue for="U_j" value="590.09802" timestamp="20"/>
          <attvalue for="U_j" value="647.70398" timestamp="21"/>
          <attvalue for="U_j" value="632.53802" timestamp="22"/>
          <attvalue for="U_j" value="624.578" timestamp="23"/>
          <attvalue for="U_j" value="607.88702" timestamp="24"/>
          <attvalue for="U_j" value="619.13098" timestamp="25"/>
          <attvalue for="U_j" value="613.45697" timestamp="26"/>
          <attvalue for="U_j" value="629.729" timestamp="27"/>
          <attvalue for="U_j" value="652.00299" timestamp="28"/>
          <attvalue for="U_j" value="669.79401" timestamp="29"/>
          <attvalue for="U_j" value="669.91498" timestamp="30"/>
          <attvalue for="U_j" value="720.23798" timestamp="31"/>
          <attvalue for="U_j" value="729.21698" timestamp="32"/>
        </attvalues>
      </node>
    </nodes>
    <edges>
      <edge id="0" source="1" target="68">
        <spells>
          <spell timestamp="1"/>
          <spell timestamp="2"/>
          <spell timestamp="3"/>
          <spell timestamp="4"/>
          <spell timestamp="5"/>
          <spell timestamp="6"/>
          <spell timestamp="7"/>
          <spell timestamp="8"/>
          <spell timestamp="9"/>
          <spell timestamp="10"/>
          <spell timestamp="11"/>
          <spell timestamp="12"/>
          <spell timestamp="13"/>
          <spell timestamp="14"/>
          <spell timestamp="15"/>
          <spell timestamp="16"/>
          <spell timestamp="17"/>
          <spell timestamp="18"/>
          <spell timestamp="19"/>
          <spell timestamp="20"/>
          <spell timestamp="21"/>
          <spell timestamp="22"/>
          <spell timestamp="23"/>
          <spell timestamp="24"/>
          <spell timestamp="25"/>
          <spell timestamp="26"/>
          <spell timestamp="27"/>
          <spell timestamp="28"/>
          <spell timestamp="29"/>
          <spell timestamp="30"/>
          <spell timestamp="31"/>
          <spell timestamp="32"/>
        </spells>
        <attvalues>
          <attvalue for="SourceIFS" value="1"/>
          <attvalue for="TargetIFS" value="68"/>
          <attvalue for="homelender" value="1"/>
          <attvalue for="hostlender" value="0"/>
          <attvalue for="P_i" value="1.97668209106579" timestamp="1"/>
          <attvalue for="P_i" value="2.09973506315347" timestamp="2"/>
          <attvalue for="P_i" value="2.45925191319754" timestamp="3"/>
          <attvalue for="P_i" value="2.44443902662783" timestamp="4"/>
          <attvalue for="P_i" value="3.256050218777" timestamp="5"/>
          <attvalue for="P_i" value="2.75911924327635" timestamp="6"/>
          <attvalue for="P_i" value="3.28524477348046" timestamp="7"/>
          <attvalue for="P_i" value="3.52308824682166" timestamp="8"/>
          <attvalue for="P_i" value="3.34988547655007" timestamp="9"/>
          <attvalue for="P_i" value="5.33517008172755" timestamp="10"/>
          <attvalue for="P_i" value="7.70901140699" timestamp="11"/>
          <attvalue for="P_i" value="4.59105096242078" timestamp="12"/>
          <attvalue for="P_i" value="8.94643814138128" timestamp="13"/>
          <attvalue for="P_i" value="7.95512330602099" timestamp="14"/>
          <attvalue for="P_i" value="6.98936499773256" timestamp="15"/>
          <attvalue for="P_i" value="5.97434468248027" timestamp="16"/>
          <attvalue for="P_i" value="6.3861588718766" timestamp="17"/>
          <attvalue for="P_i" value="6.44929660284288" timestamp="18"/>
          <attvalue for="P_i" value="7.52850887999418" timestamp="19"/>
          <attvalue for="P_i" value="7.23242802433737" timestamp="20"/>
          <attvalue for="P_i" value="7.97557179022324" timestamp="21"/>
          <attvalue for="P_i" value="7.77556328187205" timestamp="22"/>
          <attvalue for="P_i" value="8.46044965799564" timestamp="23"/>
          <attvalue for="P_i" value="8.36764445724987" timestamp="24"/>
          <attvalue for="P_i" value="8.33114702842274" timestamp="25"/>
          <attvalue for="P_i" value="8.60846847712861" timestamp="26"/>
          <attvalue for="P_i" value="8.53781427852672" timestamp="27"/>
          <attvalue for="P_i" value="9.54441157392156" timestamp="28"/>
          <attvalue for="P_i" value="9.16162519255681" timestamp="29"/>
          <attvalue for="P_i" value="10.0198608469522" timestamp="30"/>
          <attvalue for="P_i" value="11.6946345320956" timestamp="31"/>
          <attvalue for="P_i" value="12.1275693589466" timestamp="32"/>
          <attvalue for="P_j" value="4.2197230866417" timestamp="1"/>
          <attvalue for="P_j" value="4.39979865403761" timestamp="2"/>
          <attvalue for="P_j" value="5.09944303568611" timestamp="3"/>
          <attvalue for="P_j" value="4.75951266623695" timestamp="4"/>
          <attvalue for="P_j" value="6.74614818094151" timestamp="5"/>
          <attvalue for="P_j" value="5.90370476095378" timestamp="6"/>
          <attvalue for="P_j" value="6.16731602857685" timestamp="7"/>
          <attvalue for="P_j" value="6.40177184997296" timestamp="8"/>
          <attvalue for="P_j" value="6.34498308192999" timestamp="9"/>
          <attvalue for="P_j" value="9.65231295042528" timestamp="10"/>
          <attvalue for="P_j" value="12.25503997104" timestamp="11"/>
          <attvalue for="P_j" value="7.82244387831453" timestamp="12"/>
          <attvalue for="P_j" value="26.6013033463938" timestamp="13"/>
          <attvalue for="P_j" value="24.0867186859617" timestamp="14"/>
          <attvalue for="P_j" value="26.0494838022298" timestamp="15"/>
          <attvalue for="P_j" value="22.1308204788774" timestamp="16"/>
          <attvalue for="P_j" value="23.5125800714768" timestamp="17"/>
          <attvalue for="P_j" value="22.8328327346315" timestamp="18"/>
          <attvalue for="P_j" value="22.4371824267271" timestamp="19"/>
          <attvalue for="P_j" value="21.7533351493028" timestamp="20"/>
          <attvalue for="P_j" value="23.4611805226208" timestamp="21"/>
          <attvalue for="P_j" value="24.1073271769498" timestamp="22"/>
          <attvalue for="P_j" value="24.2600587916962" timestamp="23"/>
          <attvalue for="P_j" value="23.8876641912834" timestamp="24"/>
          <attvalue for="P_j" value="24.2559320808014" timestamp="25"/>
          <attvalue for="P_j" value="24.9918099390084" timestamp="26"/>
          <attvalue for="P_j" value="25.9986422731048" timestamp="27"/>
          <attvalue for="P_j" value="29.4840672433113" timestamp="28"/>
          <attvalue for="P_j" value="28.1991190097385" timestamp="29"/>
          <attvalue for="P_j" value="29.6504789309234" timestamp="30"/>
          <attvalue for="P_j" value="34.1623195155579" timestamp="31"/>
          <attvalue for="P_j" value="32.4725022173784" timestamp="32"/>
          <attvalue for="weight" value="15.552" timestamp="1"/>
          <attvalue for="weight" value="17.763" timestamp="2"/>
          <attvalue for="weight" value="21.594" timestamp="3"/>
          <attvalue for="weight" value="22.077999" timestamp="4"/>
          <attvalue for="weight" value="33.412998" timestamp="5"/>
          <attvalue for="weight" value="31.862" timestamp="6"/>
          <attvalue for="weight" value="36.749001" timestamp="7"/>
          <attvalue for="weight" value="39.776001" timestamp="8"/>
          <attvalue for="weight" value="38.702999" timestamp="9"/>
          <attvalue for="weight" value="59.393002" timestamp="10"/>
          <attvalue for="weight" value="76.550003" timestamp="11"/>
          <attvalue for="weight" value="40.255001" timestamp="12"/>
          <attvalue for="weight" value="163.498" timestamp="13"/>
          <attvalue for="weight" value="140.843" timestamp="14"/>
          <attvalue for="weight" value="151.75101" timestamp="15"/>
          <attvalue for="weight" value="121.87" timestamp="16"/>
          <attvalue for="weight" value="136.412" timestamp="17"/>
          <attvalue for="weight" value="134.06" timestamp="18"/>
          <attvalue for="weight" value="132.306" timestamp="19"/>
          <attvalue for="weight" value="128.366" timestamp="20"/>
          <attvalue for="weight" value="151.959" timestamp="21"/>
          <attvalue for="weight" value="152.48801" timestamp="22"/>
          <attvalue for="weight" value="151.52299" timestamp="23"/>
          <attvalue for="weight" value="145.21001" timestamp="24"/>
          <attvalue for="weight" value="150.17599" timestamp="25"/>
          <attvalue for="weight" value="153.314" timestamp="26"/>
          <attvalue for="weight" value="163.72099" timestamp="27"/>
          <attvalue for="weight" value="192.237" timestamp="28"/>
          <attvalue for="weight" value="188.87601" timestamp="29"/>
          <attvalue for="weight" value="198.633" timestamp="30"/>
          <attvalue for="weight" value="246.05" timestamp="31"/>
          <attvalue for="weight" value="236.795" timestamp="32"/>
        </attvalues>
      </edge>
    </edges>
  </graph>
</gexf>
```

**Comment by eduramiba** [comment]
Fixed for 0.9.2, please test with the latest snapshot: https://github.com/gephi/gephi#nightly-builds
